gems/aws-sdk-core/spec/aws-sdk-core/credentials_provider_chain_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+module AWS::SDK::Core
+  describe CredentialsProviderChain do
+    subject { described_class }
+
+    context '.call' do
```

Comment:
should this be a describe instead of context?

Reply:
Yes